CMakeLists.txt

```diff
@@ -192,33 +186,34 @@ if (USE_LIBCUDF)
     #       instead of hardcoding paths to includes and to libraries, like below
     target_include_directories(meta-cudf-parser-1 PUBLIC
                                /usr/local/include
-                               /usr/local/include/libcudf/libcudacxx
+                               $ENV{CONDA_PREFIX}/include
     )
     target_include_directories(meta-cudf-parser-1 AFTER PUBLIC
-                               /opt/conda/envs/rapids/include
+        $ENV{CONDA_PREFIX}/include
     )
     target_link_libraries(meta-cudf-parser-1 PRIVATE
                           cudf
                           -L/usr/local/cuda/lib64 -L/usr/local/lib
-                          -L/opt/conda/envs/rapids/lib
+                          -L$ENV{CONDA_PREFIX}/lib
```

Comment:
universal instead of hardfixed

Reply:
also /opt/conda/envs/rapids doesn't exist in a default conda installation of rapids

Reply:
RAPIDS docker images used to have some of necessary includes and libs in `/opt/conda/envs/rapids/` - while Dockerfile does not set `CONDA_PREFIX` environment variable. That's why it is here: for `sudo docker build .` to work.

Adding the `$ENV{CONDA_PREFIX}/lib` part is good, but I'd wait with removing `/opt/conda/envs/rapids/` (see split commits).